main.py

```
@@ -0,0 +1,37 @@
+from flask import Flask, request
+from utils import *
+
+app = Flask(__name__)
+
+
+@app.route("/dsvw")
+def dsvw():
+    init()
```

Ruff (0.11.9)

`init` may be undefined, or defined from star imports

(F405)